src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -28,6 +29,14 @@
       <File Include="$(CoreCLRBuildIntegrationDir)*" TargetPath="build" />
       <File Include="$(CoreCLRILCompilerDir)netstandard\*" TargetPath="tools/netstandard" />
     </ItemGroup>
+
+    <!-- exclude unix-y native symbols from ilc package (they are included in symbols package) -->
```

Comment:
Does this mean that we are still including `ilc.pdb` in the ilc package on Windows?

Reply:
Yup, I modeled it after @MichalStrehovsky's comment:

> Does this remove things like Runtime.ServerGC.pdb (on Windows) or System.Private.CoreLib.pdb (on all platforms)?
>
> We deliberately want them in the package.

If we want to apply it for Windows as well, we have a way to distinguish native vs. managed pdb in nearby context.

Reply:
The distintion should be aot runtime vs. tools. The symbols should be stripped for tools (both managed and native), but kelp for runtime bits.

The tools are under `tools` directory in the package. Can we do the filter based on the directory?